process/transaction/process.go

```diff
@@ -128,29 +125,18 @@ func (txProc *txProcessor) processTxFee(tx *transaction.Transaction, acntSnd *st
 		return nil, nil
 	}
 
-	cost := big.NewInt(0)
-	cost = cost.Mul(big.NewInt(0).SetUint64(tx.GasPrice), big.NewInt(0).SetUint64(tx.GasLimit))
-
-	txDataLen := int64(len(tx.Data))
-	txProc.mutTxFee.RLock()
-	minTxFee := big.NewInt(0).SetUint64(txProc.economicsFee.MinGasLimit())
-	minTxFee.Mul(minTxFee, big.NewInt(0).SetUint64(txProc.economicsFee.MinGasPrice()))
-
-	minFee := big.NewInt(0)
-	minFee.Mul(big.NewInt(txDataLen), big.NewInt(0).SetUint64(txProc.economicsFee.MinGasPrice()))
-	minFee.Add(minFee, minTxFee)
-	txProc.mutTxFee.RUnlock()
-
-	if minFee.Cmp(cost) > 0 {
-		return nil, process.ErrNotEnoughFeeInTransactions
+	err := txProc.economicsFee.CheckTxHandler(tx)
```

Comment:
rename the function name, as it does not say what is does in the background.

Reply:
renamed to CheckValidityTxValues